packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -945,3 +976,418 @@ class _RenderTheatre extends RenderBox with ContainerRenderObjectMixin<RenderBox
     ];
   }
 }
+
+// This API is inspired by the flutter_portal package by @rrousselGit and
```

Comment:
Given there are so few, do you think it makes sense to credit the other contributors as well? I am not sure myself because looking at the implementation, I do realize that it does differ significantly from the code we wrote.